src/folding/nova/circuits.rs

```diff
@@ -648,13 +639,14 @@ pub mod tests {
         let cmT = Projective::rand(&mut rng);
 
         // compute the challenge natively
-        let r = AugmentedFCircuit::<Projective, TestFCircuit<Fr>>::get_challenge_native(
+        let r_bits = ChallengeGadget::<Projective>::get_challenge_native(
             &poseidon_config,
             u_i.clone(),
             U_i.clone(),
             cmT,
         )
         .unwrap();
+        let r = Fr::from_bigint(BigInteger::from_bits_le(&r_bits)).unwrap();
```

Comment:
I was using unwrap because we're in a test here, with the idea that if this line fails it breaks the test. Not sure I'm understanding the reason to use expect here

Reply:
Sure, did not see the test module. My bad.